README.md

```diff
@@ -44,6 +44,7 @@ To get started with local development, create a file in the root of the repo nam
 ```bash
 GITHUB_PERSONAL_API_TOKEN=<your_github_api_token>
 BUGZILLA_ENDPOINT=http://localhost:3090
+PORT=5000
```

Comment:
Do we use `PORT` outside of `.neutrinorc.js`? If not, then we can remove `PORT` from here and have the value be hard-coded. Also remove `NODE_ENV : 'development'` from `.neutrinorc.js` since webpack does that for us when we do `"dotenv webpack-dev-server -- --mode development",`.

```diff
--- a/.neutrinorc.js
+++ b/.neutrinorc.js
@@ -28,14 +28,12 @@ module.exports = {
         favicon: './src/static/favicon.png',
       },
       devServer: {
-        port: process.env.PORT,
+        port: 5000,
         historyApiFallback: { disableDotRule: true },
       },
       env: {
         GITHUB_PERSONAL_API_TOKEN : '',
         BUGZILLA_ENDPOINT : 'http://localhost:3090',
-        NODE_ENV : 'development',
-        PORT: 5000,
       },
       babel: {
         plugins: [
```

Reply:
Yes, we dont use it anywhere else apart from the .neutrinorc.js.. Sure!